crates/iota/src/name_commands.rs

```diff
@@ -110,6 +110,14 @@ pub enum NameCommand {
         /// The coin to use for payment. If not provided, selects the first coin
         /// with enough balance.
         coin: Option<ObjectID>,
+        /// The address or alias to which the domain will point. If the flag is
+        /// specified without a value, the current active address will be used.
+        #[arg(long, require_equals = true, default_missing_value = "true", num_args = 0..=1)]
+        set_target_address: Option<String>,
+        /// Set the reverse lookup domain. If the flag is specified without a
+        /// value, the registered domain will be used.
+        #[arg(long, require_equals = true, default_missing_value = "true", num_args = 0..=1)]
+        set_reverse_lookup: Option<String>,
```

Comment:
I don't think we should actually allow passing a different domain, because then it has nothing to do with the register command anymore

Reply:
I mean it's possible to do with the other command so it kind of makes sense to allow it here

Reply:
I disagree, `set_target_address` makes perfect sense because it's tied to the domain being created, `set_reverse_lookup` with another domain just doesn't make sense

Reply:
To me this also doesn't make sense, should be just bool to set reverse lookup or not, but not providing another domain here

Reply:
but then if you want to register the reverse lookup to some other domain you have to call the other function

Reply:
Yeah and you should, because it has absolutely nothing to do with the current registration